config/locales/controllers/admin/en.yml

```diff
@@ -5,3 +5,10 @@ en:
       update_success: Extensions have been updated.
     spaces_controller:
       update_success: Space has been updated.
+    users_controller:
+      update_success: User has been updated.
```

Comment:
this is causing the e2e test to fail - the sentences here include a period that wasn't in the original code

```
assert expected User has been updated. to equal User has been updated
```

Reply:
Ah, great catch! I've updated this.